omnibus_build.sh

```diff
@@ -30,6 +24,22 @@ rm -rf /opt/$PROJECT_NAME/*
 
 builtin cd $PROJECT_DIR
 
+# Install the gems we need, with stubs in bin/
+bundle update # Make sure to update to the latest version of omnibus-software
+
+# Loading release information from release.json
+vars=`bundle exec ruby load_realease.rb`
+
+# Export variables from the release.json
+IFS="
```

Comment:
nit: I think I prefer
```
IFS=$'\n'
```
because it's more explicit (but feel free to ignore)

Also, although it doesn't affect the script in the following lines, it might be the right thing to restore IFS to the default behavior:
```
IFS_DEFAULT=$IFS
IFS=$'\n'
for line in $vars;
do
...
done
IFS=$IFS_DEFAULT
```